.circleci/config.yml

```diff
@@ -94,17 +96,18 @@ jobs:
           name: 🏗️ build dash
           command: |
             . venv/bin/activate && mkdir packages
+            set -eo pipefail
             # build main dash
             git clone --depth 1 https://github.com/plotly/dash.git dash-main
             cd dash-main && pip install -e .[dev] --progress-bar off && python setup.py sdist && mv dist/* ../packages/
             cd dash-renderer && npm ci && npm run build
             python setup.py sdist && mv dist/* ../../packages/ && cd ../..
-            # build dcc
-            npm ci && npm run build && python setup.py sdist && mv dist/* ./packages
             # build html
             git clone --depth 1 https://github.com/plotly/dash-html-components.git
             cd dash-html-components && npm ci && npm run build
             python setup.py sdist && mv dist/* ../packages && cd .. && ls -la packages
+            # build dcc
+            npm ci && npm run build && python setup.py sdist && mv dist/* ./packages
```

Comment:
No specific reason. Just feels more natural to build dcc last as we are in dcc..